pkgs/development/python-modules/intbitset/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, six
+, nose
+}:
+buildPythonPackage rec {
+  pname = "intbitset";
+  version = "2.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "44bca80b8cc702d5a56f0686f2bb5e028ab4d0c2c1761941589d46b7fa2c701c";
+  };
+
+  # fixes compilation on aarch64 and determinism (uses -march=core2 and
+  # -mtune=native)
+  postPatch = ''
+    sed '/extra_compile_args/d' -i setup.py
```

Comment:
Use `substituteInPlace`.

Reply:
How do I wield it to delete the entire line matching that search? That's why I used `sed` -- it was easier to do that.

(e.g. there's more than just `extra_compile_args` that needs deleting)

Reply:
Oh I see. Then it's fine.

Reply:
However, I see that this will not work after the next update. Maybe using `patches` is better?

Reply:
Ah, yeah, good idea. Done.